src/website/app/demos/Flex/examples/Flex/direction.js

```diff
@@ -32,7 +32,7 @@ Additionally, their flow can be reversed.
     the <em key={0}>visual</em> order while maintaining
     the <em key={1}>semantic</em> order. This can affect keyboard users because
     of a non-intuitive tab order as well as the order of content as announced by
-    assistive technology (AT).
+    assistive technologies.
```

Comment:
I think maybe here we should use the word "identified" rather than "announced" since not all assistive technologies involve sound... thoughts?

Reply:
+1

Reply:
"recognized" perhaps